### PR TITLE
DropDownButton Demo: Set Button Template for DropDownButton

### PR DESCRIPTION
**Pull request by jdvictoria**

### CODE REVIEW

File: NetCoreDemos/Views/DropDownButton/Overview.cshtml [hunk 1]
```diff
@@ -154,3 +168,76 @@
         }
     }
 </script>
+
+<style>
+    .demo-container .dx-fieldset:first-child {
+        width: 500px;
+    }
+
+    .custom-color-picker {
```
Comment (AlexanderMoiseev) [inline NetCoreDemos/Views/DropDownButton/Overview.cshtml]: i believe we alredy have this style. please check it. and why did not you use css file for this demo?

File: NetCoreDemos/Views/DropDownButton/Overview.cshtml [hunk 2]
```diff
@@ -22,16 +22,30 @@
...
         <div class="dx-field-value">
+            @{
+                var employeeName = "Olyvia Peyton";
+                var employeeIcon = Url.Content("~/images/employees/51.png");
+            }
             @(Html.DevExtreme().DropDownButton()
                 .SplitButton(true)
-                .Text("Sandra Johnson")
-                .Icon(Url.Content("~/images/gym/coach-woman.png"))
+                .Text(employeeName)
```
Comment (AlexanderMoiseev) [inline NetCoreDemos/Views/DropDownButton/Overview.cshtml]: [minor] lets leave it inlined as it was before

File: MVCDemos/Content/DemosStyles/DropDownButton/Overview.css [hunk 3]
```diff
@@ -17,6 +17,53 @@
     font-size: 18px;
 }
 
-#text {
```
Comment (AlexanderMoiseev) [inline MVCDemos/Content/DemosStyles/DropDownButton/Overview.css:20]: we have this style in our js demos, we should not remove it

**Review by vladaskorohodova — DISMISSED** [review]

File: MVCDemos/Views/DropDownButton/Overview.cshtml [hunk 4]
```diff
@@ -19,26 +19,38 @@
 
     <div class="dx-field">
         <div class="dx-field-label">
-            Custom main button action
+            Custom template and actions
         </div>
         <div class="dx-field-value">
+            @{
+                var employeeName = "Olivia Peyton";
```
Comment (AlexanderMoiseev) [inline MVCDemos/Views/DropDownButton/Overview.cshtml:26]: lets just inline it

File: MVCDemos/Views/DropDownButton/Overview.cshtml [hunk 5]
```diff
@@ -19,26 +19,38 @@
...
                 .DisplayExpr("Text")
                 .KeyExpr("Value")
                 .UseSelectMode(false)
                 .DataSource(Model.ProfileSettings)
                 .OnButtonClick("profileButton_click")
                 .OnItemClick("profileItem_click")
+                .Template(@<text>
+                    <div class="button-img-container">
+                        <div class="button-img-indicator"></div>
+                        <img class="button-img" src="@Url.Content("~/images/employees/51.png")" alt="imageAlt" />
```
Comment (AlexanderMoiseev) [inline MVCDemos/Views/DropDownButton/Overview.cshtml]: why did path to 'images' folder change? it used to be like ../../Content/Images

File: MVCDemos/Views/DropDownButton/Overview.cshtml [hunk 6]
```diff
@@ -19,26 +19,35 @@
...
                 .DisplayExpr("Text")
                 .KeyExpr("Value")
                 .UseSelectMode(false)
                 .DataSource(Model.ProfileSettings)
                 .OnButtonClick("profileButton_click")
                 .OnItemClick("profileItem_click")
+                .Template(@<text>
+                    <div class="button-img-container">
+                        <div class="button-img-indicator"></div>
+                        <img class="button-img" src="../../Content/Images/employees/51.png" alt="imageAlt" />
```
Comment (AlexanderMoiseev) [inline MVCDemos/Views/DropDownButton/Overview.cshtml]: imageAlt is not descriptive, lest rename it to employee, same for netCore demo

File: NetCoreDemos/Views/DropDownButton/Overview.cshtml [hunk 7]
```diff
@@ -19,26 +19,35 @@
...
                 .DisplayExpr("Text")
                 .KeyExpr("Value")
                 .UseSelectMode(false)
                 .DataSource(Model.ProfileSettings)
                 .OnButtonClick("profileButton_click")
                 .OnItemClick("profileItem_click")
+                .Template(@<text>
+                    <div class="button-img-container">
+                        <div class="button-img-indicator"></div>
+                        <img class="button-img" src="@Url.Content("~/images/employees/51.png")" alt="imageAlt" />
```
Comment (AlexanderMoiseev) [inline NetCoreDemos/Views/DropDownButton/Overview.cshtml]: imageAlt - > employee

File: MVCDemos/Views/DropDownButton/Overview.cshtml [hunk 8]
```diff
@@ -19,26 +19,35 @@
...
                 .DisplayExpr("Text")
                 .KeyExpr("Value")
                 .UseSelectMode(false)
                 .DataSource(Model.ProfileSettings)
                 .OnButtonClick("profileButton_click")
                 .OnItemClick("profileItem_click")
+                .Template(@<text>
+                    <div class="button-img-container">
+                        <div class="button-img-indicator"></div>
+                        <img class="button-img" src="../../Content/Images/employees/51.png" alt="employee" />
```
Comment (AlexanderMoiseev) [inline MVCDemos/Views/DropDownButton/Overview.cshtml:37]: i can see that other images added as a resource to the project's file, like  /<Content Include="Content\Images\employees\50.png" //>" . Shouldn't we add them as well?  but please check project builds after adding it

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Pull request merged**